lib/fluent/plugin/out_copy.rb

```diff
@@ -55,10 +57,15 @@ def process(tag, es)
         }
         es = m
       end
-
+      success = Array.new(outputs.size)
       outputs.each_with_index do |output, i|
         begin
-          output.emit_events(tag, @copy_proc ? @copy_proc.call(es) : es)
+          if i > 0 && success[i - 1] && @ignore_if_prev_successes[i]
```

Comment:
If ignore_if_prev_success is specified in first `<store>` section, 
it may be better to raise an error.

Reply:
Then, `ignore_error` is specified in all `store` without `ignore_if_prev_success` it may be better to warn.

Reply:
> If ignore_if_prev_success is specified in first `<store>` section,
> it may be better to raise an error.

This should be fixed at https://github.com/fluent/fluentd/pull/3287/commits/2f5a6e1450b3e3dbbb6d3e856e76dfb4224118b5.

Reply:
> Then, `ignore_error` is specified in all `store` without `ignore_if_prev_success` it may be better to warn.

This also should be fixed at https://github.com/fluent/fluentd/pull/3287/commits/bdbaef288c73868d414becfbbb47bf663265af07.